### ISSUE TITLE
Can spatial domain guided 3D slice alignment be used for 10X data

### ISSUE THREAD

**Issue by HaixJiang** [zhoux85/STAligner#22]
Can spatial domain guided 3D slice alignment be used for 10X data?

**Comment by zhoux85** [comment]
Yes, In principle, it can be applied to any platform as long as there are common landmark spatial domains between slices, which has clear tissue structure to guide the alignment.
